src/components/pages/Cases/CaseTable.js

```diff
@@ -318,7 +318,7 @@ export default function CaseTable(props) {
       ...getColumnSearchProps('case_origin_state'),
     },
     {
-      title: 'Filed within Year',
+      title: 'Filed within One Year',
       dataIndex: 'filed_within_year',
       key: 'filed_within_year',
```

Comment:
I know we had a PR that hasn't been merged yet that fixes the naming of variables, but should 322, 323, and 324 have been changed?

Reply:
Looks good to me, seems like a simple change but would check filed_within_year.

Reply:
I actually don't think those were changed in that pull request as we were searching for and revising "filed_in_one_year" and not "within". I'm not sure if these are something different though or if maybe we overlooked them. Those were also changed to "check_for_one_year" so I'm not sure if that would need to be applied here as well.